connectors/webhook/src/main/java/io/camunda/connector/inbound/signature/strategy/HMACEncodingStrategy.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.inbound.signature.strategy;
+
+import io.camunda.connector.api.inbound.webhook.WebhookProcessingPayload;
+import java.io.IOException;
+
+public interface HMACEncodingStrategy {
```

Comment:
Suggestion: we can implement a sealed hierarchy here like with [`WebhookAuthorizationHandler`](https://github.com/camunda/connectors/blob/2589015a805272da8688610b4d829fcb8cb28e3b/connectors/webhook/src/main/java/io/camunda/connector/inbound/authorization/WebhookAuthorizationHandler.java#L22) and its subclasses.

```suggestion
public sealed interface HMACEncodingStrategy {
```